pandas/tests/extension/test_arrow.py

```diff
@@ -3133,6 +3133,60 @@ def test_arrow_floordiv():
     tm.assert_series_equal(result, expected)
 
 
+def test_arrow_floordiv_large_values():
+    # GH 55561
+    a = pd.Series(
+        [
+            1425801600000000000,
```

Comment:
Thanks could you just test 1 of these values? (we don't need all of them to reproduce the error)

Reply:
Done.